README.md

```diff
@@ -4,10 +4,14 @@ At the sametime, it also helps me to know you a little bit more :)
 
 ## Brief self-introduction [max 500 words]
 
+Hello everyone. I'm Meng-Jiun Chiou, a senior student of NCTU ECE. I'm not sure if I'm the only NCTU student joining this lecture, however I hope to get involved with you happily. I haven't found my teammate yet, so I'll appreciate if anyone could let me in. Although I'm ECE student, I am rather interested in CS so that I have attended some fundamental classes of CS. My hobby is jogging, table tennis and watching Japan dramas and animations.
```

Comment:
Dear Meng-Jiun,
I'm Sumit J, also a student from NCTU EECS. very glad to hear you and I would like to join you as your teammate if you don't mind! I think as we both are from same school so it would be convenient for us to work together.
looking forward to hear from you...:)
Sumit J